assets/sass/_variables.scss

```diff
@@ -49,9 +49,9 @@ $form-divider-color: #e8e8e8;
 $facebook-icon-bg-color: #304879; // fb brand color - #3B5998
 $facebook-label-bg-color: #406094;
 $facebook-border-color: #375685;
-$google-icon-bg-color: #d43722; // google brand color - #DC4E41
-$google-label-bg-color: #de4937;
-$google-border-color: #b63c33;
+$google-icon-bg-color: #fff;
+$google-label-bg-color: $button-bg-color;
+$google-border-color: darken($button-bg-color, 22%);
```

Comment:
Can we actually get a hex here? It's easier to categorize and switch to DS when we have it.

Reply:
I'm not too worry about that now since it's probably got an redesign before DS hit SIW.